ceph.spec.in

```diff
@@ -131,6 +131,7 @@ BuildRequires:	python-pecan
 BuildRequires:	python-requests
 BuildRequires:	python-virtualenv
 BuildRequires:	python-werkzeug
+BuildRequires:	socat
```

Comment:
@smithfarm yes just for make check. Feel free to make it part of a conditional. Not it's also could be a conditional for debs and FreeBSD too.